admin/sql/CreateViews.sql

```diff
@@ -13,36 +13,6 @@ CREATE OR REPLACE VIEW release_event AS
         FROM release_unknown_country
     ) as q;
 
--- First, construct a table (recursively) of parent -> descendant connections
--- for areas, including an array of the path (the 'descendants' array).
-
--- Then, find the shortest path to each type of parent by joining to area,
--- distinct on descendant, type, and order by the length of the array of descendants.
-
--- link type de7cc874-8b1b-3a05-8272-f3834c968fb7 is the area-area 'parent of' relation
-CREATE OR REPLACE VIEW area_containment AS
-        WITH RECURSIVE area_descendants AS (
```

Comment:
See https://www.postgresql.org/docs/9.5/static/queries-with.html for some insight on why this was slow:

> However, the other side of this coin is that the optimizer is less able to push restrictions from the parent query down into a WITH query than an ordinary subquery. The WITH query will generally be evaluated as written, without suppression of rows that the parent query might discard afterwards.